package.json

```diff
@@ -11,7 +11,8 @@
   },
   "dependencies": {
     "node-fetch": "^2.1.1",
-    "probot": "^6.0.0"
+    "probot": "^6.0.0",
+    "sshpk": "^1.14.1"
```

Comment:
Can you add the `querystring` dependency as well?

Reply:
IIRC `querystring` is a Node.JS library. Is it normal to add an explicit dependency on libraries that are part of the standard distribution of Node.JS? I don't actually want the `npm` package by the same name.

Reply:
Oh ok, nevermind then 👍